arroyo-console/src/routes/pipelines/JobGraph.tsx

```diff
@@ -40,16 +58,32 @@ function PipelineGraphNode({
 
 export function PipelineGraph({
   graph,
+  metrics,
   setActiveOperator,
   activeOperator,
 }: {
   graph: JobGraph;
+  metrics?: JobMetricsResp;
   setActiveOperator: (op: string) => void;
   activeOperator?: string;
 }) {
   const nodeTypes = useMemo(() => ({ pipelineNode: PipelineGraphNode }), []);
 
   const nodes = graph.nodes.map(node => {
+    let operatorBackpressure = 0;
+    if (metrics) {
+      // reduce the subtasks backpressure to an operator-level backpressure
+      const nodeMetrics = metrics.metrics[node.nodeId];
+      const backPressureMetrics = Object.entries(nodeMetrics.subtasks).map(kv => {
```

Comment:
I think this is causing errors before metrics are available as the pipeline is starting up:
> TypeError: Cannot read properties of undefined (reading 'subtasks')

Reply:
This is fixed now.